.circleci/config.yml

```diff
@@ -269,31 +229,66 @@ jobs:
               -destination 'platform=iOS Simulator,name=iPhone 11' \
               -scheme ApplePlatformsIntegrationMobileTests
           working_directory: test/ApplePlatformsIntegrationTestApp
+
+  build-apple-runtime:
+    <<: *apple_defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: v3-repo-{{ .Environment.CIRCLE_PR_NUMBER }}
+      - run:
+          name: Set up workspace
+          command: mkdir -p /tmp/hermes/output
+      - run:
+          name: Install dependencies
+          command: |
+            brew install cmake
+            sudo gem install cocoapods
+      - run:
+          name: Build the iOS frameworks
+          command: ./utils/build-ios-framework.sh
+      - run:
+          name: Build the Mac frameworks
+          command: ./utils/build-mac-framework.sh
+      - run:
+          name: Build the CLI for Mac
+          command: cmake --build ./build_macosx --target github-cli-release
       - save_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          key: v3-repo-{{ .Environment.CIRCLE_PR_NUMBER }}
```

Comment:
Changes to cache key are temporary to avoid rebuilding the whole framework (which takes a long time). I'd like to change this cache key to something that actually indicates whether there were changes that need to be rebuilt or not.

Reply:
Yeah I'm cool with that. Let's revert it to something stricter when the new setup is getting stable.

Reply:
I like that plan!